web/src/locales/en.json

```diff
@@ -13,6 +13,10 @@
   "settings-modal": {
     "title": "Settings"
   },
+  "announcement-modal": {
+    "title": "Introducing a visual update to our app!",
+    "content": "We've revamped the colors representing electricity sources, bringing them closer to industry standards."
+  },
```

Comment:
As a general FYI, I don't think it we need to do translations for short-lived messages like this one :)